Main.py

```diff
@@ -64,18 +69,62 @@ def wait_for_tweet_and_buy():
         time.sleep(4)
     return symbol
 
-def sell_at_peak(symbol):
-    if symbol in binance_coins:
-        return
-    if symbol in bittrex_coins:
-        return
-    return
 
+def get_price_bittrex(bittrex, symbol):
+    market = 'BTC-' + symbol.upper()
+    ticker = bittrex.get_ticker(market)
+    return ticker['result']['Last']
+
+def get_price_binance(binance, symbol):
+    market = symbol.upper() + 'BTC'
+    prices = binance.get_orderbook_tickers()
+    price = list(filter(lambda m: m['symbol'] == market, prices))
+    return float(price[0]['askPrice'])
+
+
+def sell_at_peak(state, symbol):
+    if state['bittrex']:
+        last_price_bittrex = get_price_bittrex(bittrex, symbol)
+    if state['binance']:
+        last_price_binance = get_price_binance(binance, symbol)
+
+    while state['bittrex'] or state['binance']:
+        time.sleep(CHECK_TIMER)
+
+        if state['bittrex']:
+            cur_price_bittrex = get_price_bittrex(bittrex, symbol)
+            delta = (cur_price_bittrex - last_price_bittrex) / last_price_bittrex
+            last_price_bittrex = cur_price_bittrex
+
+            utils.print_and_write_to_logfile('CHECKING PRICE ON BITTREX')
+            utils.print_and_write_to_logfile("CURRENT PRICE: " + str(last_price_bittrex))
+            utils.print_and_write_to_logfile("DELTA: " + str(delta) + '\n')
+
+            if delta < SELL_THRESHOLD:
+                utils.print_and_write_to_logfile("THRESHOLD REACHED | SELLING ON BITTREX...")
+
+                # sell TODO
+
+                state['bittrex'] = False
 
-bought_symbol = wait_for_tweet_and_buy()
+        if state['binance']:
+            cur_price_binance = get_price_binance(binance, symbol)
+            delta = (cur_price_binance - last_price_binance) / last_price_binance
+            last_price_binance = cur_price_binance
 
-sell_at_peak(bought_symbol)
+            utils.print_and_write_to_logfile('CHECKING PRICE ON BINANCE')
+            utils.print_and_write_to_logfile("CURRENT PRICE: " + str(last_price_binance))
+            utils.print_and_write_to_logfile("DELTA: " + str(delta) + '\n')
 
+            if delta < SELL_THRESHOLD:
+                utils.print_and_write_to_logfile("THRESHOLD REACHED | SELLING ON BINANCE...")
 
+                # sell TODO
 
+                state['bittrex'] = False
 
+# # # # #  MAIN  # # # # #
+utils.print_and_write_to_logfile('STARTING BUY PHASE')
+#bought_symbol = wait_for_tweet_and_buy() TODO uncomment buy
+utils.print_and_write_to_logfile('STARTING SELL PHASE')
+sell_at_peak(state, 'REQ')
```

Comment:
hard coded in some random symbol for testing but will be `bought_symbol` once its done